lib/web-client.js

```diff
@@ -129,7 +131,15 @@ class WebClient {
 
   _getCollection(ns) {
     const namespace = toNS(ns);
-    const db = this.stitchClient.service('mongodb', 'mongodb1').db(namespace.database);
+    let db;
+    if (this.model.connectionType === CONNECTION_TYPE_VALUES.STITCH_ATLAS
```

Comment:
logic should be:

- for stitch atlas: hardcode to `mongodb-atlas`
- for stitch on-prem: use `this.model.stitchServiceName`